### ISSUE TITLE
Documentation for C58862 Error Code appears missing

### ISSUE THREAD

**Issue by mbarnet** [dotnet/docs#28792]
using ...
namespace somenamespace {
[IBindable(BindableSupport.Yes,Serializable()]
public record class ClassName() : IDisposable , IXmlSerializable
{
     public_ ClassName()
     {
          //throws a **C58862** Error even with class properties initialized.
     }
...
}}

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bbb051ee-f020-66ba-c601-4a08ad32ef61
* Version Independent ID: 58a19505-ac46-c5c8-12cc-fe87b6aa18a7
* Content: [Sorry, we don't have specifics on this C# error](https://docs.microsoft.com/en-us/dotnet/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error?f1url=%3FappId%3Droslyn%26k%3Dk(CS8862))
* Content Source: [docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md](https://github.com/dotnet/docs/blob/main/docs/csharp/misc/sorry-we-don-t-have-specifics-on-this-csharp-error.md)
* Product: **dotnet-csharp**
* Technology: **csharp-diagnostics**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

**Comment by BillWagner** [comment]
Thanks for raising this @mbarnet 

This is an area where we've fallen behind in the docs.

Here's a smaller example:

```csharp
public record ClassName(int num)
{
   //Compiler generates C58862 Error even with class properties initialized.
    public ClassName()
    {
       this.num = 0;
    }
}
```

This is because the compiler's static analysis requires that the generated primary constructor is called from all user-written constructors in that class.

**Comment by BillWagner** [comment]
Fixed in #35324
